### PR TITLE
Set parent region on imported data source records.

### PR DESCRIPTION
**Pull request by kiyote33**

### CODE REVIEW

**Comment by kiyote33** [comment]
Hi @joshreisner, Sub-domains are inherently supported in the region hierarchy. To achieve what you are suggesting (which I don't believe is what CER really desires), simply import the Berlin data source with no region selected. Once imported, use the Region utility to assign Germany as the parent to the Berlin, Frankfurt (Oder), and Near Berlin regions. That will give you the desired region hierarchy: 
image

**Comment by joshreisner** [comment]
@kiyote33 i think it's safe to say CER desires the regions to be the way they currently are on their site: Germany > Berlin > Kreuzberg.

our goal should be to enable them to achieve that without editing regions post-import

**Comment by kiyote33** [comment]
FYI: I built the current Germany region hierarchy by hand for them so they would have Berlin meetings displaying under Germany while we worked on a resolution to their problem. But I don't think it would be that difficult to add the existing hierarchy to the code, so I will add that in...

**Comment by kiyote33** [comment]
Supports sub_sub_region:
image

**Review by joshreisner — APPROVED** [review]
works for me, thanks!

File: includes/ajax.php [hunk 1]
```diff
@@ -339,7 +339,7 @@ function tsml_ajax_geocode()
 
 	$meetings	= get_option('tsml_import_buffer', []);
 	$errors		= [];
-	$limit		= 25;
+	$limit		= 5;
```
Comment (tech2serve) [inline includes/ajax.php:342]: This is a pretty big change, and I might have missed a discussion, but will we see a performance change on large imports?

Apologies if I’m out in left field as I’m working to catch up.

**Review by tech2serve — COMMENTED** [review]
I’ve not tested the PR in it’s entirety, so I’m not going to provide explicit approval, just wanted to make sure I understood the `$limit` change.

File: includes/ajax.php (hunk 1 above)
Comment (kiyote33) [reply includes/ajax.php]: Oversight! Now fixed...

**Comment by kiyote33** [comment]
Fixes issue #809

**Pull request merged**